sourcegen-model/src/main/java/io/micronaut/sourcegen/model/MethodDef.java

```diff
@@ -477,6 +488,30 @@ public MethodDefBuilder addStatements(@NonNull Collection<StatementDef> newState
             return this;
         }
 
+        /**
+         * Add throw expressions to the method.
+         *
+         * @param types The types that this method throws
+         * @return The builder
+         */
+        @NonNull
+        public MethodDefBuilder addThrows(@NonNull ClassTypeDef... types) {
```

Comment:
Maybe lets use `throws`

Reply:
I know it would be better, but `throws` is a Java keyword

Reply:
Right 🥹